resources/codegen_inputs/ift.rs

```diff
@@ -250,4 +251,47 @@ table TablePatch {
 flags u8 TablePatchFlags {
   REPLACE_TABLE = 0b01,
   DROP_TABLE = 0b10,
+}
+
+/// [Glyph Keyed Patch](https://w3c.github.io/IFT/Overview.html#glyph-keyed)
+table GlyphKeyedPatch {
+  format: Tag,
+  #[skip_getter]
+  #[compile(0)]
+  _reserved: u32,
+  flags: GlyphKeyedFlags,
+  #[traverse_with(skip)]
+  compatibility_id: CompatibilityId,
+  max_uncompressed_length: u32,
+  #[count(..)]
+  brotli_stream: [u8],
+}
+
+flags u8 GlyphKeyedFlags {
+  NONE = 0b0,
+  WIDE_GLYPH_IDS = 0b1,
+}
+
+/// [GlyphPatches](https://w3c.github.io/IFT/Overview.html#glyphpatches)
+#[read_args(flags: GlyphKeyedFlags)]
+table GlyphPatches {
+  glyph_count: u32,
+  table_count: u8,
+
+  #[count($glyph_count)]
+  #[read_with($flags)]
+  #[traverse_with(skip)]
+  #[compile(skip)] // TODO remove this once write fonts side is implemented.
```

Comment:
are we trying to compile this table? If not maybe it makes more sense to skip the whole table, instead of generating write-fonts code that is going to be incorrect. You can add `#[skip_font_write]` to skip just this table, although it might also break other tables that reference it..

Reply:
In this first pass I'm not going to implement compilation, but I will be adding eventually. So far I've just been using #[compile(skip)] on anything that causes any issues with the plan being to go back and fix it all up once I get to implementing compilation.